wbooks/src/app/components/CustomButton/styles.tsx

```diff
@@ -0,0 +1,32 @@
+import { StyleSheet } from 'react-native';
+import Colors from '@constants/colors';
+import Fonts from '@constants/fonts';
+
+const styles = StyleSheet.create({
+  primaryContainer: {
+    height: 44,
+    backgroundColor: Colors.primary,
+    borderRadius: 100
+  },
+  primaryText: {
+    ...Fonts.PrimaryButton,
+    paddingVertical: 10,
+    alignSelf: 'center',
+    textTransform: 'uppercase'
+  },
+  secondaryContainer: {
+    height: 44,
+    backgroundColor: Colors.white,
+    borderRadius: 100,
+    borderWidth: 1,
+    borderColor: Colors.primary
+  },
+  secondaryText: {
+    ...Fonts.SecondaryButton,
+    paddingVertical: 10,
+    alignSelf: 'center',
+    textTransform: 'uppercase'
+  }
```

Comment:
You are repeating the same styles in container and texts, use a basic style and only add the different ones

for example:
```
  primaryContainer: {
    height: 44,
    backgroundColor: Colors.primary,
    borderRadius: 100
  },
  secondaryContainer: {
    borderWidth: 1,
    borderColor: Colors.primary
  },
```
Same with the texts